### PR TITLE
[ISSUE #859] Add unit tests for EventMesh SDK for Java

### PR DESCRIPTION
**Pull request by wallezhang**
Fixes ISSUE #859.

### Motivation

Add missing unit tests

### Modifications

Add unit tests for `eventmesh-sdk-java` module.

### Documentation

- [ ] Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    public void setUp() throws Exception {
+        eventMeshGrpcConsumer = new EventMeshGrpcConsumer(EventMeshGrpcClientConfig.builder().build());
+        eventMeshGrpcConsumer.init();
+        FieldSetter.setField(eventMeshGrpcConsumer, eventMeshGrpcConsumer.getClass().getDeclaredField("consumerClient"),
+            consumerClient);
+        FieldSetter.setField(eventMeshGrpcConsumer,
+            eventMeshGrpcConsumer.getClass().getDeclaredField("consumerAsyncClient"), consumerAsyncClient);
+        FieldSetter.setField(eventMeshGrpcConsumer,
+            eventMeshGrpcConsumer.getClass().getDeclaredField("heartbeatClient"), heartbeatClient);
+        when(consumerClient.subscribe(any())).thenReturn(Response.getDefaultInstance());
```
Comment (ruanwenjun) [inline eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java:79]: Use reflect to create test object is not a good practice, When someone changes the field name, he cannot find out the problem in compile.
You can try to change the scope of field in `consumerClient`. The same to other test file.

**Review by ruanwenjun — CHANGES_REQUESTED** [review]

File: eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java (hunk 1 above)
Comment (wallezhang) [reply eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java:79]: Yes, if someone change the field name, error will only be thrown when the test case is run. Change the field scope to `protected` will resolve the problem. I have another idea, what about we create a setter method with protected modification and `VisibleForTest` comment. Like this:

```java
private ConsumerServiceBlockingStub consumerClient;

/**
*VisibleForTest
**/
protected void setConsumerClient(ConsumerServiceBlockingStub consumerClient) {
    this.consumerClient = consumerClient;
}
```

Comment (ruanwenjun) [reply eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java:79]: I would like to change the field to default or protected, no need to add setter method, in fact, we shouldn't add a method to modify this field.

Comment (wallezhang) [reply eventmesh-sdk-java/src/test/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumerTest.java:79]: OK, I try to fix it.

**Comment by qqeasonchen** [comment]
@wallezhang please fix the conflicts.

**Comment by wallezhang** [comment]
> @wallezhang please fix the conflicts.

Done.

**Review by qqeasonchen — APPROVED** [review]
LGTM

**Comment by qqeasonchen** [comment]
@ruanwenjun please check if the change you requested is resoved, then can merge it.

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Pull request merged**